docs/development/workflow/maintainer_workflow.rst

```diff
@@ -171,6 +171,25 @@ One can control the bot by making comments on the pull request:
   If you wish to run the pre-commit check first in CI without running Actions,
   use ``[skip actions]`` or ``[actions skip]`` in your commit message.
 
+.. _benchmarks:
+
+Benchmarks
+==========
+
+If a pull request explicitly deals with a performance improvement, maintainers should
+use the ``benchmark`` label to run a comparative benchmark on the pull request. This
+will run a benchmark comparing the ``HEAD`` commit of the pull request branch with
+the main branch. The logs are uploaded as a action artefact and can be viewed
+in the Actions tab.
+
+Maintainers should also run the benchmark if they suspect the code change might have
+performance implications. The benchmark action takes approximately 25 minutes to
+complete.
+
+It is important to note that the benchmarking on Github Actions will be flakey and
```

Comment:
Which one is more correct? 🤷‍♀️ 
```suggestion
It is important to note that the benchmarking on Github Actions will be flaky and
```

Reply:
No idea. This is taking me to the dark linguistics corners of the internet. It seems like either is fine. But flaky is more used now.